### PR TITLE
[81_4] Implement multi-line display for numerous tab pages

### PR DESCRIPTION
**Pull request by Minzihao**
## What
See in [https://github.com/XmacsLabs/mogan/pull/1961](https://github.com/XmacsLabs/mogan/pull/1961)
## How to test your changes?
Create numerous tab pages using `Ctrl+N`, and then maximize/minimize the window to view changes.

### CODE REVIEW

**Comment by Minzihao** [comment]
我相信现在不是匿名邮箱了

**Comment by Minzihao** [comment]
我相信现在不是匿名邮箱了

**Comment by da-liii** [comment]
合并一下最新的branch-1.2分支，ci就可以通过了

File: src/Plugins/Qt/QTMTabPage.cpp [hunk 1]
```diff
@@ -11,18 +11,21 @@
 
 #include "QTMTabPage.hpp"
 
+// The minimum width of a single tab page (in pixels).
```
Comment (da-liii) [inline src/Plugins/Qt/QTMTabPage.cpp:14]: resolved

File: src/Plugins/Qt/QTMTabPage.cpp [hunk 2]
```diff
@@ -31,10 +34,109 @@ QTMTabPage::QTMTabPage (url p_url, QAction* p_title, QAction* p_closeBtn,
...
+                      tabWidth, m_rowHeight);
+    accumWidth+= tabWidth;
+    accumTab+= 1;
+  }
+
+  adjustHeight (rowCount);
+}
+
+void
+QTMTabPageContainer::removeAllTabPages () {
```
Comment (da-liii) [inline src/Plugins/Qt/QTMTabPage.cpp:87]: resolved

**Comment by da-liii** [comment]
LGTM (我已经检查了我的review全部通过了）

**Comment by Minzihao** [comment]
All checks have passed @da-liii

**Review by Oyyko — APPROVED** [review]
LGTM

**Pull request merged**